AirCasting/SDSync/ViewsFlow/SDSyncComplete/SDSyncCompleteViewModel.swift

```diff
@@ -0,0 +1,14 @@
+// Created by Lunar on 06/12/2021.
+//
+
+import Foundation
+
+protocol SDSyncCompleteViewModel: ObservableObject {
```

Comment:
I wanted to move `tabSelection` in here too, but I think it's not that easy... Let's skip